share/ext/luajit/build_luajit.sh

```diff
@@ -202,64 +207,21 @@ case $1 in
 			echo "****************************************************"
 			file $PREFIX/bin/$CONF_TARGET/luajit-${DEFOLD_ARCH}
 			echo "****************************************************"
-			export DEFOLD_ARCH="64"
-			export XCFLAGS="${XCFLAGS} ${ARCH_FLAGS} ${COMMON_XCFLAGS}"
-
-			export HOST_CC="clang"
-			export HOST_CFLAGS="${COMMON_XCFLAGS} ${ARCH_FLAGS} -m64 -I."
-			export HOST_ALDFLAGS="-m64"
-			export TARGET_LDFLAGS="-m64"
-
-			echo "Building $CONF_TARGET ($DEFOLD_ARCH) with '$XCFLAGS'"
-			set -e
-			make -j8
-			make install
-			mv $PREFIX/bin/$CONF_TARGET/${TARGET_FILE} $PREFIX/bin/$CONF_TARGET/luajit-${DEFOLD_ARCH}
-
-			echo "****************************************************"
-			file $PREFIX/bin/$CONF_TARGET/luajit-${DEFOLD_ARCH}
-			echo "****************************************************"
-
-			# remove symlink. we don't need it, and it doesn't extract under git-bash
-			rm -v $PREFIX/bin/$CONF_TARGET/luajit
-			set +e
-		}
-		;;
-	arm64-linux)
-		export TARGET_SYS=Linux
-		function cmi_make() {
-			export ARCH_FLAGS="--target=aarch64-unknown-linux-gnu"
 
-			export DEFOLD_ARCH="32"
-			export XCFLAGS="-DLUAJIT_TARGET=LUAJIT_ARCH_ARM -DLUAJIT_DISABLE_GC64 ${COMMON_XCFLAGS}"
-
-			export HOST_CC="clang"
-			export HOST_CFLAGS="${COMMON_XCFLAGS} -I."
-			export HOST_LDFLAGS=""
-			export TARGET_CFLAGS="${ARCH_FLAGS}"
-			export TARGET_LDFLAGS="${ARCH_FLAGS}"
-
-			echo "Building $CONF_TARGET ($DEFOLD_ARCH) with '$XCFLAGS'"
-			set -e
-			make -j8
-			make install
-			mv $PREFIX/bin/$CONF_TARGET/${TARGET_FILE} $PREFIX/bin/$CONF_TARGET/luajit-${DEFOLD_ARCH}
-			make clean
-			set +e
-
-			echo "****************************************************"
-			file $PREFIX/bin/$CONF_TARGET/luajit-${DEFOLD_ARCH}
-			echo "****************************************************"
+			if [ "arm64-linux" == "${PLATFORM}" ]; then
+				XCFLAGS="-DLUAJIT_TARGET=LUAJIT_ARCH_ARM64 ${XCFLAGS}"
+			fi
 
 			export DEFOLD_ARCH="64"
-			export XCFLAGS="-DLUAJIT_TARGET=LUAJIT_ARCH_ARM64 ${COMMON_XCFLAGS}"
+			export XCFLAGS="${ARCH_FLAGS} ${COMMON_XCFLAGS}"
 
 			export HOST_CC="clang"
-			export HOST_CFLAGS="${COMMON_XCFLAGS} ${ARCH_FLAGS} -m64 -I."
-			export HOST_ALDFLAGS="-m64"
-			export TARGET_LDFLAGS="-m64"
+			export HOST_CFLAGS="${COMMON_XCFLAGS} -m64 -I."
+			export HOST_LDFLAGS="-m64"
+			export TARGET_CFLAGS="-m64 ${ARCH_FLAGS}"
+			export TARGET_LDFLAGS="-m64 ${ARCH_FLAGS}"
```

Comment:
Another change. we want to use the host settings for the host build, and the target architecture for the target build.
Another topic for the CI build workflow is to add *Unit Tests* for this!